doc/formatter.md

```diff
@@ -92,6 +165,16 @@ InterTokenInfo before each token.
 
 A lot of other details are traced by running with `-v=4 --alsologtostderr`.
 
+Selective output from Verible modules can be controled via [glog] environment
```

Comment:
"controlled"

Also, this might be a better fit somewhere in general developer documentation.